src/Layout/QueryParamContext.tsx

```diff
@@ -642,7 +649,7 @@ function ErrorCard({
     if (showErrorCard) {
       setTimeout(() => {
         setShowErrorCard(false);
-      }, 3000);
+      }, 5000);
```

Comment:
1. Should the timeout be longer, or perhaps we should remove the timeout entirely?

Reply:
https://github.com/Plant-for-the-Planet-org/planet-donations/pull/351#discussion_r1064306326

Reply:
Timeout left as is over here.